[2022-07-29 09:02:39,982] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: Demo.data_uploading scheduled__2022-07-28T09:00:00+00:00 [queued]>
[2022-07-29 09:02:39,995] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: Demo.data_uploading scheduled__2022-07-28T09:00:00+00:00 [queued]>
[2022-07-29 09:02:39,995] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-07-29 09:02:39,995] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2022-07-29 09:02:39,996] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-07-29 09:02:40,021] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): data_uploading> on 2022-07-28 09:00:00+00:00
[2022-07-29 09:02:40,027] {standard_task_runner.py:52} INFO - Started process 2407 to run task
[2022-07-29 09:02:40,031] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'Demo', 'data_uploading', 'scheduled__2022-07-28T09:00:00+00:00', '--job-id', '57', '--raw', '--subdir', 'DAGS_FOLDER/demo.py', '--cfg-path', '/tmp/tmp8zsqhj6f', '--error-file', '/tmp/tmpu21gbe8o']
[2022-07-29 09:02:40,032] {standard_task_runner.py:80} INFO - Job 57: Subtask data_uploading
[2022-07-29 09:02:40,126] {task_command.py:371} INFO - Running <TaskInstance: Demo.data_uploading scheduled__2022-07-28T09:00:00+00:00 [running]> on host 949e67af4713
[2022-07-29 09:02:40,228] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=Demo
AIRFLOW_CTX_TASK_ID=data_uploading
AIRFLOW_CTX_EXECUTION_DATE=2022-07-28T09:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-07-28T09:00:00+00:00
[2022-07-29 09:02:40,230] {python.py:173} INFO - Done. Returned value was: 0
[2022-07-29 09:02:40,264] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=Demo, task_id=data_uploading, execution_date=20220728T090000, start_date=20220729T090239, end_date=20220729T090240
[2022-07-29 09:02:40,323] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-29 09:02:40,413] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
